etrago/tools/execute.py

```diff
@@ -213,12 +242,29 @@ def iterate_lopf(
     """
 
     args = etrago.args
+    path = args["csv_export"]
+    lp_path = args["lpfile"]
 
-    if dispatch_disaggregation:
+    if (
+        args["temporal_disaggregation"]["active"] is True
+        and etrago.conduct_dispatch_disaggregation is False
+    ):
+        if not args["csv_export"] is False:
+            path = path + "/temporally_reduced"
+
+        if not args["lpfile"] is False:
+            lp_path = lp_path[0:-3] + "_temporally_reduced.lp"
```

Comment:
Is it really needed to store these .lp files? 
It created a lot of additional data